init-ci.ps1

```diff
@@ -80,6 +81,24 @@ if ([string]::IsNullOrEmpty($DemoTeamRegistry)) {
     throw
   }
 }
+
+if ([string]::IsNullOrEmpty($SitecoreRegistry)) {
+  # if it wasn't passed as a parameter, let's try to find it in environment
+  $SitecoreRegistry = $env:INTERNAL_SITECORE_DOCKER_REGISTRY
+  if ($null -eq $SitecoreRegistry) {
+    # Environment variable not found. Try to set it using demo team function.
+    Set-DemoEnvironmentVariables
+    refreshenv
+  }
+
+  # Retry
+  $SitecoreRegistry = $env:INTERNAL_SITECORE_DOCKER_REGISTRY
+  if ($null -eq $SitecoreRegistry) {
+    # Environment variable not found. Use public Sitecore Docker registry.
+    $SitecoreRegistry = "scr.sitecore.com/"
+  }
+}
+
```

Comment:
Please remove the retry. If SitecoreRegistryis not passed or INTERNAL_SITECORE_DOCKER_REGISTRY is not found in the environment, simply output a warning.